src/nimiq-style.css

```diff
@@ -124,18 +125,24 @@
 }
 
+.icon-keyfile-locked { /* gray closed lock */
+    width: 9rem;
+    height: 11.75rem;
+    background-image: url('data:image/svg+xml,<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 72 94" style="enable-background:new 0 0 72 94;" xml:space="preserve"><path d="M43,74.04c-0.02-1.15,0.39-2.23,1.17-3.05c0.74-0.79,1.76-1.26,2.83-1.31C47.81,63.65,52.87,59,58.99,59 c2.61,0,5.03,0.86,7.01,2.3V23.83c0-0.49-0.19-0.95-0.54-1.3l-22-22C43.12,0.19,42.65,0,42.17,0H1.83C0.83,0,0,0.82,0,1.83v84.33 C0,87.18,0.83,88,1.83,88H43V74.04z M40.33,3.67l22,22h-22V3.67z M33,67c-0.08,0-0.16-0.01-0.23-0.04C20.01,62.09,19,52.68,19,40.9 c0-0.31,0.2-0.58,0.49-0.66c0.1-0.03,9.87-2.77,13.02-6.04c0.25-0.26,0.71-0.26,0.96,0c3.16,3.27,12.93,6.01,13.03,6.04 C46.8,40.31,47,40.59,47,40.9c0,11.79-1.01,21.19-13.77,26.06C33.16,66.99,33.08,67,33,67z" opacity="0.15" fill-rule="evenodd" clip-rule="evenodd" /><path d="M70.7,72.66h-2.6v-1.33c0-5.15-4.09-9.33-9.11-9.33c-5.02,0-9.11,4.19-9.11,9.33v1.33h-2.6 c-0.72,0-1.3,0.59-1.28,1.33v18.67C45.97,93.4,46.56,94,47.28,94H70.7c0.72,0,1.3-0.6,1.3-1.33V74C72,73.26,71.42,72.66,70.7,72.66z M60.29,82.93v4.4c0,0.74-0.58,1.33-1.3,1.33c-0.72,0-1.3-0.6-1.3-1.33v-4.4c-0.76-0.47-1.3-1.29-1.3-2.26 c0-1.47,1.17-2.67,2.6-2.67c1.44,0,2.6,1.2,2.6,2.67C61.59,81.64,61.05,82.46,60.29,82.93z M65.49,72.67H52.48v-1.33 c0-3.68,2.92-6.67,6.51-6.67c3.59,0,6.51,2.99,6.51,6.67V72.67z" opacity="0.15" fill-rule="evenodd" clip-rule="evenodd" /></svg>');
+}
+
 /***************
```

Comment:
icon-keyfile-secure, icon-keyfile-insecure, icon-keyfile-locked werden 2x definiert?!

Reply:
Where else is it defined?

Reply:
in line 48-51

Reply:
Yes, this is how all icons are defined. At the top are general styles for all icons, and then the specific sizes and actual backgrounds are defined individually. I think you may have been confused by how GitHub collapses the changes. Extend the file to see how it works.

Reply:
Ok, but weight and height do get overwritten. Anyway

Reply:
Yes, because most icons have the same dimensions, so it makes sense to declare it once for all. But some icons are special and have other sizes, so we overwrite it.

You are welcome to suggest a different method.